components/dialog/UgTutorDialog.tsx

```diff
@@ -51,13 +55,18 @@ const decisionColourMap = {
   [Decision.PENDING]: 'bg-amber-200'
 }
 
-const UgTutorForm: FC<UgTutorFormProps> = ({ data, readOnly, setCurrentTab }) => {
+const UgTutorForm: FC<UgTutorFormProps> = ({ data, reviewerLogin, readOnly, setCurrentTab }) => {
   const { applicant, internalReview } = data
   const [outcomes, setOutcomes] = useState(data.outcomes)
```

Comment:
I can't find another use of `setOutcomes` other than the one in `useEffect` on line 66. Perhaps this should be 

```suggestion
  const outcomes = useMemo(() => data.outcomes, [data.outcomes])
```

And now, here I am asking: do we _even_ need this `outcomes` variable?

Reply:
The outcomes are changed by the `Dropdown` and `TextField.Root`